### PR TITLE
ncm-metaconfig: httpd: add missing wsgi and some other configuration options

### PR DESCRIPTION
**Pull request by stdweird**
Adding OIDC to openstack keystone implies generating the whole config.

Requires panc >= 10.7 (uses equals in annotation).

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan [hunk 1]
```diff
@@ -386,15 +387,33 @@ type httpd_perl_handler = {
...
-type httpd_wsgi_iportscript = {
+type httpd_wsgi_importscript = {
     "path" : string
     "process" ? string
     "application" ? string
 };
 
+type httpd_wsgi_daemonprocess = {
+    "name" : string
+    @{converted in list of key=value}
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan]: panc on Jenkins is falling over the equals in this annotation, time to update to 10.7?

Comment (stdweird) [reply ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan]: sure, but then there's no way back.you can move the milestone to a release where we can expect that 10.7 is ok to run. this is a parse error, not a compilation error, so afaik, as soon as this temnplate in your repo, you must be running 10.7

Comment (jrha) [reply ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan]: Hmm, could you remove the `=` then?

**Comment by stdweird** [comment]
retest this please

**Pull request merged**